core/common/src/main/java/alluxio/metrics/EventCounter.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.metrics;
+
+import org.apache.log4j.AppenderSkeleton;
+import org.apache.log4j.Level;
+import org.apache.log4j.spi.LoggingEvent;
+
+/**
+ * A log4J Appender that simply counts logging events in four levels:
+ * FATAL, ERROR WARN and INFO. The class name is used in log4j.properties
+ */
+public class EventCounter extends AppenderSkeleton {
+  private static final int FATAL = 0;
+  private static final int ERROR = 1;
+  private static final int WARN = 2;
+  private static final int INFO = 3;
+  private static EventCounter.EventCounts sCount = new EventCounter.EventCounts();
+
+  /**
+   * The constructor of EventCounter.
+   */
+  public EventCounter() {
+  }
+
+  /**
+   * Get the number of fatal log.
```

Comment:
According to javaDoc convention, we use 3rd person (descriptive) 
can you help change the `Get` to `Gets` in the method description, similar for all other methods

Reply:
ok,I will change it.